extension/flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisExecutionOptions.java

```diff
@@ -105,6 +111,25 @@ public Builder setStreamLoadProp(Properties streamLoadProp) {
             return this;
         }
 
+        public Builder setJsonFormat() {
+            this.streamLoadProp.setProperty("format", "json");
+            this.streamLoadProp.setProperty("strip_outer_array", "true");
+            return this;
+        }
+
+        public Builder setMergeType(MergeType mergeType) {
+            this.streamLoadProp.setProperty("merge_type", mergeType.toString());
+            return this;
+        }
+
+        public Builder setDeleteLabel(String deleteLabel, String value) {
+            if (!MergeType.MERGE.toString().equals(this.streamLoadProp.getProperty("merge_type"))) {
+                return this;
+            }
+            this.streamLoadProp.setProperty("delete", deleteLabel + "=" + value);
```

Comment:
the delete flag can be any expressions. For example:

`delete: ifnull(column1, 2) > 3`

So could you find a way to make it more flexible?

Reply:
ok, I will check it

Reply:
this.streamLoadProp.setProperty("delete", deleteflag);
How about it? If the delete flag can be any expressions, the user set the value by "deletaflag".

Reply:
looks good to me~